crafters/image/ImageReader/tests/test_imagereader.py

```diff
@@ -34,3 +34,12 @@ def test_io_buffer():
     test_doc = crafter.craft(buffer=image_buffer.getvalue(), uri=None)
     assert test_doc['blob'].shape == (img_size, img_size, 3)
     np.testing.assert_almost_equal(test_doc['blob'], np.array(img).astype('float32'))
+
+
+def test_io_buffer_metadata():
+    crafter = ImageReader()
+    import requests
+    img_url='https://raw.githubusercontent.com/x4nth055/pythoncode-tutorials/master/ethical-hacking/image-metadata-extractor/image.jpg'
```

Comment:
if this `image` is OpenSource, I would suggest to download it and add it inside the `tests` folder. I think is better not to depend on texternal Urls that may eventually change and may harm the test

Reply:
hi @JoanFM  I have taken the image from this repo: https://github.com/x4nth055/pythoncode-tutorials which has an MIT licence, but I am not entirely sure about the [image.jpg](https://github.com/x4nth055/pythoncode-tutorials/tree/master/ethical-hacking/image-metadata-extractor), because of that I was afraid to put it on the repo.

Reply:
I think we may find some proper test image data that has metadata and can be valid for us.

Reply:
Also it would be good to add mocking tests to ensure that every option of the metadata info is tested, for instance the check if data is of type bytes